tests/functional/test_config.py

```diff
@@ -297,6 +297,25 @@ def test_deployments(networks_connected_to_tester, owner, vyper_contract_contain
     assert deployment.address == instance.address
 
 
+def test_deterministic_deployments(
```

Comment:
```suggestion
def test_deployments_deterministic(
```

I know this sounds backwards, but it lets us do this in `pytest`:

```
-k test_deployments
```

to run all deployments related tests at once, which is something i do a lot: run all related tests at once when sseeing if broke something